src/data/shared/header-menu.js

```diff
@@ -128,6 +128,13 @@ const headerMenu = [
         },
         to: 'https://ebpf.io/slack',
       },
+      {
+        title: {
+          en: 'eBPF User Group',
+          'fr-fr': 'eBPF User Group',
```

Comment:
```suggestion
          en: 'eBPF Meetup',
          'fr-fr': 'eBPF Meetup',
```

Reply:
I'm afraid I have to disagree with this. 

Not only meetups can happen in the User Group, but a meetup is just one part of the User Group, other events happen in the User Group and this is how I've seen other OSS communities name it. 

It shouldn't be eBPF Meetup

Reply:
I spoke with Daniel about this too and he wants to go with Meetup because user group sounds old fashioned

Reply:
alright then! sounds good